src/core/Rule.ml

```diff
@@ -233,9 +235,17 @@ type extract_spec = {
   dst_lang : Xlang.t;
   (* e.g., $...BODY, $CMD *)
   extract : MV.mvar;
+  extract_rules_paths : extract_rules_paths option;
   transform : extract_transform;
 }
 
+(* SR wants to be able to choose rules to run on
+   Behaves the same as paths *)
+and extract_rules_paths = {
```

Comment:
those are not paths to  rules right now, just rule ids, so maybe extract_rule_ids
or you name it that way to allow glob at some point?